index.bs

```diff
@@ -1914,47 +1914,39 @@ WebAuthn supports multiple attestation types:
     ECDAA-Issuer (see [[FIDOEcdaaAlgorithm]]).
 
 
-### Generating an Attestation Object ### {#generating-an-attestation-object}
+<h4 id="generating-an-attestation-object" algorithm>Generating an Attestation Object</h4>
```

Comment:
Adding the [algorithm] attribute makes Bikeshed check that I've used all the variables I declare, unless I specifically mark them as `<var ignore>`.

Reply:
ah ha, innaresting.